api/v1alpha1/hostedcluster_types.go

```diff
@@ -266,6 +266,10 @@ const (
 	IBMCloudPlatform PlatformType = "IBMCloud"
 )
 
+const (
+	PortierisImageAnnotation = "hypershift.openshift.io/portieris-image"
```

Comment:
nit: please put this with the rest the annotation constants above and add a brief description for it

Reply:
you got it done!